.vscode/settings.json

```diff
@@ -1,3 +1,2 @@
 {
-	"workbench.colorTheme": "Kimbie Dark"
```

Comment:
Hello @dodo721 

I find it helpful to have vs-code in different colours for different projects. It helps avoid confusion when switching between editors.
Having vscode settings in git makes this setting team-wide.
Let's not edit settings. But if you don't like it, we could maybe move the file out of git, and it could become a personal setting.

Daniel

Reply:
Ah that was committed by mistake - i'll update it in the next commit